### ISSUE TITLE
Can't release old claim

### ISSUE THREAD

**Issue by kimihiro64** [lbryio/lbry#707]
Awhile back, I had an overturned claim with a substantial amount of LBC in it, but because of a bug at the time, I was unable to release that particular claim. And now I've forgotten how to look up and release that claim to see if it's been fixed. Pointers on listing and releasing existing claims with daemon commands? Reporting as an issue in case it's still bugged.

About
Your copy of LBRY is up to date.

daemon (lbrynet)	0.11.0
wallet (lbryum)	2.7.25
interface	0.12.0
Platform	Linux (Linux-4.4.0-75-generic-x86_64-with-Ubuntu-16.04-xenial)
Installation ID	3n1vtReZyaeUpTweiXPuVF7ncX1HVAKj3mJp9Lhpns71kbh6c9yv8yDzHenokJWRfc

**Comment by tzarebczan** [comment]
Please join us on slack to ask questions like these instead of opening issues. Here is a YouTube tutorial on how to abandon. The commands are in the api guide too. We will be making this possible with the GUI in the future. 

https://youtu.be/l1eqoE9xNsg

**Comment by kimihiro64** [comment]
satan@satan-Aspire-V5-591G:~/Downloads/LBRY Daemon$ ./lbrynet-cli claim_list_mine
WARNING:lbrynet.conf:Ignoring invalid conf file setting: wallet_type
WARNING:lbrynet.conf:Ignoring invalid conf file setting: requested_first_run_credits
WARNING:lbrynet.conf:Ignoring invalid conf file setting: start_lbrycrdd
[]

Now I can't even get the claim id >.>

**Comment by tzarebczan** [comment]
Are you running the daemon? Are you on the old wallet type, i.e wallet.dat as opposed to default_wallet? When is your orignal install from? Might need help from @jack here, but give him some time as it's the weekend.

**Comment by kimihiro64** [comment]
Original install was one of the first alpha clients. But the claim issue must've happened somewhere around 0.6 or 0.5, I /think/ the transaction happened on the newer lbryum wallet.

**Comment by kimihiro64** [comment]
Address that holds the claim: https://explorer.lbry.io/#!/address/bPLRZUG7Kodc4aqeAkcF2gJL1WGDFRepKj

Transaction ID: https://explorer.lbry.io/#!/transaction/5ecc82698582e10e2f0f6346c5aabafe26d24d5aebd529daf31e13256355b2a1

Claim ID: c0de6d96466986e5a1b6d3c1c52c6712d1f12049

But when I try to abandon, says claim not found. Likewise claim_list_mine returns empty. That address is in my wallet though. Issue is likely somewhere in here:

https://github.com/lbryio/lbryum/blob/ce78bbec809f9fe866c2eae3d3d024ae75e22d2d/lib/wallet.py#L1000

Possibly https://github.com/lbryio/lbryum/blob/ce78bbec809f9fe866c2eae3d3d024ae75e22d2d/lib/wallet.py#L1018

**Comment by kimihiro64** [comment]
It looks like the bug is caused by the fact that the claim was stored incorrectly. When I filter get_name_claims to only look for address bPLRZUG7Kodc4aqeAkcF2gJL1WGDFRepKj, it has no txos or txis returned from self.get_addr_io(addr).

**Comment by kimihiro64** [comment]
self.txo.get(tx_hash, {}) for tx_hash 5ecc82698582e10e2f0f6346c5aabafe26d24d5aebd529daf31e13256355b2a1 returns only {u'bTxsfS8mucMEGmZVHcequTp861XnaEbtit': [[0, 947200, False]]}. In other words, the actual address with the claim amount bPLRZUG7Kodc4aqeAkcF2gJL1WGDFRepKj was not listed as an address, even though its own history is tied to that tx_hash. It would seem my 1510 literally fell off the block chain.

**Comment by tzarebczan** [comment]
Did @wallermadev help you out? Can we close this?

**Comment by kimihiro64** [comment]
No, not fully resolved. I'm not really sure how to solve this honestly. I can try and fake the output to get it to try and recognize it was my claim, but not sure if the rest of the clients will accept that as a valid transaction.

**Comment by kimihiro64** [comment]
I hacked the code many, many times to force it to submit the abandon claim with (I hope) valid attributes, but it came back with this:

{
"reason": "error: The transaction was rejected by network rules.(16: mandatory-script-verify-flag-failed (Script failed an OP_EQUALVERIFY operation))\n[0100000001a1b2556325131ef3da29d5eb5a4dd226febaaac546630f2f0ee182856982cc5e000000006b483045022100fd09a5c9f6d753d677347507da2f7c3b7aed21c31a2a3f46e741a9f95c39e4a1022075531f571990cf25749100892d36ceb42c6f6fe9039ceeaa7a7d7bcc3c3b77fa0121038206dbea8c234c95d90431868524bb81f240355a679ac4f0b4fe4856983dc0c8ffffffff0100740e00000000001976a914b3dc3966069b30380b9369eeab5d37f701afee3f88ac00000000]",
   "success": false
}

**Comment by kimihiro64** [comment]
Was able to successfully abandon the transaction. Part of the bug was that the output address did not get stored in the 'txo' section of the wallet (I fixed this manually). The other part was that the nOut for this claim is 1, when lbry assumes nOut of 0 for claims. In this case nOut = 0 was the transaction fee, so I had to allow it in the code to use 0 in order to pass the sanity checks, and then change it back to 1 when signing the transaction so it would use the right input. The bad-txns-in-belowout error from network was because the transaction fee address nOut = 0 obviously didn't have enough to cover the abandon.
